opencti-platform/opencti-graphql/src/database/xtm-obas.ts

```diff
@@ -176,19 +175,11 @@ export const getScenarioResult = async (id: string) => {
   }
   const httpClient = buildXTmOpenBasHttpClient();
   try {
-    const { data: scenario } = await httpClient.get(`/scenarios/external_reference/${id}`);
-    if (!scenario || !scenario.scenario_id) {
+    const { data: exercise } = await httpClient.get(`/scenarios/external_reference/${id}`);
```

Comment:
just to understand: this openBAS API has changed its signature and returns now an exercise instead of a scenario, right ? 
So if openBAS and openCTI are not both updated to the latest version, this code will fail I guess ?

Reply:
Exactly, we need to have a good synchronisation between the two products

Reply:
Do we write somewhere which version of octi is compatible with which version of obas?

Reply:
We are thinking about a robuste plan to handle this.
Good point :)